### PR TITLE
add direnv

### PR DESCRIPTION
**Pull request by bbigras**

### CODE REVIEW

File: bucket/direnv.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            "url": "https://github.com/direnv/direnv/releases/download/v2.19.0/direnv.windows-amd64.exe",
+            "hash": "14654cb6e4f7bc2e5fa6f5ba0e2568956846ec0ef94c52c6fee09f245f8007cf",
+            "bin": [ [ "direnv.windows-amd64.exe", "direnv" ] ]
+        },
+        "32bit": {
+            "url": "https://github.com/direnv/direnv/releases/download/v2.19.0/direnv.windows-386.exe",
+            "hash": "085b51d85cdec3a98e40a10527a0fb8632d4282435fad7839a659b7d1b1bbf10",
+            "bin": [ [ "direnv.windows-386.exe", "direnv" ] ]
+        }
+    },
```
Comment (Ash258) [inline bucket/direnv.json]:
```suggestion
    },
    "bin": "direnv.exe"
```

File: bucket/direnv.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/direnv/direnv/releases/download/v$version/direnv.windows-amd64.exe"
```
Comment (Ash258) [inline bucket/direnv.json]:
```suggestion
                "url": "https://github.com/direnv/direnv/releases/download/v$version/direnv.windows-amd64.exe#/direnv.exe"
```

File: bucket/direnv.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+            },
+            "32bit": {
+                "url": "https://github.com/direnv/direnv/releases/download/v$version/direnv.windows-386.exe"
```
Comment (Ash258) [inline bucket/direnv.json]:
```suggestion
                "url": "https://github.com/direnv/direnv/releases/download/v$version/direnv.windows-386.exe#/direnv.exe"
```

File: bucket/direnv.json (hunk 2 above)
Comment (Ash258) [reply bucket/direnv.json]: 1. Change these autoupdate properties and run checkver with force parameter.
1. Delete architectures bin and create generic one

File: bucket/direnv.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
+{
+    "homepage": "https://direnv.net",
+    "checkver": {
+        "github": "https://github.com/direnv/direnv"
+    },
```
Comment (Ash258) [inline bucket/direnv.json]: Add description

File: bucket/direnv.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+            "hash": "14654cb6e4f7bc2e5fa6f5ba0e2568956846ec0ef94c52c6fee09f245f8007cf",
+            "bin": [ [ "direnv.windows-amd64.exe", "direnv" ] ]
+        },
+        "32bit": {
+            "url": "https://github.com/direnv/direnv/releases/download/v2.19.0/direnv.windows-386.exe",
+            "hash": "085b51d85cdec3a98e40a10527a0fb8632d4282435fad7839a659b7d1b1bbf10",
+            "bin": [ [ "direnv.windows-386.exe", "direnv" ] ]
+        }
+    },
+    "bin": "direnv.exe"
```
Comment (h404bi) [inline bucket/direnv.json]: add a comma, fix ci test

**Comment by Ash258** [comment]
Run checkver to update urls with rename fragments: https://github.com/lukesampson/scoop/pull/2985#discussion_r247810946

**Comment by bbigras** [comment]
> Run checkver to update urls with rename fragments: #2985 (comment)

Yes sorry. I wasn't using a Windows machine when I made the changes earlier today.

If everything is fine I can squash and rebase the commits if you prefer.

Thanks for you patience.

**Pull request merged**